### PR TITLE
updated SSO docs

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated SSO docs and image

### CODE REVIEW

**Review by usha-mandya — APPROVED** [review]

**Review by technicallyjosh — APPROVED** [review]

File: single-sign-on/index.md [hunk 1]
```diff
@@ -5,24 +5,24 @@ title: Configure Single Sign-on
...
+When SSO is enabled, users are redirected to your provider’s authentication page to authenticate using SSO. They cannot authenticate using their personal login credentials (Docker ID and password). Docker currently supports Service Provider Initiated SSO flow. Your users must sign into Docker Hub or Docker Desktop to initiate the SSO authentication process.
 
 Before enabling SSO in Docker Hub, administrators must work with their identity provider to configure their IdP to work with Docker Hub. Docker provides the Assertion Consumer Service (ACS) URL and the Entity ID. Administrators use this information to establish a connection between their IdP server and Docker Hub.
 
 After establishing the connection between the IdP server and Docker Hub, administrators log into the organization in Docker Hub and complete the SSO enablement process. See the section **Enable SSO in Docker Hub** for detailed instructions.
 
 To enable SSO in Docker Hub, you need the following:
 
-* **SAML 2.0**: Entity ID, ACS URL, Single Logout URL and Certificate Download URL
+* **SAML 2.0**: Entity ID, ACS URL, Single Logout URL and X.509 certificate
```
Comment (technicallyjosh) [inline single-sign-on/index.md:16]:
```suggestion
* **SAML 2.0**: Single Sign-On URL and the X.509 signing certificate
```

File: single-sign-on/index.md [hunk 2]
```diff
@@ -5,24 +5,24 @@ title: Configure Single Sign-on
...
 
-When SSO is enabled, users are redirected to your provider’s authentication page to authenticate using SSO. They cannot authenticate using their personal login credentials (Docker ID and password).
+When SSO is enabled, users are redirected to your provider’s authentication page to authenticate using SSO. They cannot authenticate using their personal login credentials (Docker ID and password). Docker currently supports Service Provider Initiated SSO flow. Your users must sign into Docker Hub or Docker Desktop to initiate the SSO authentication process.
 
 Before enabling SSO in Docker Hub, administrators must work with their identity provider to configure their IdP to work with Docker Hub. Docker provides the Assertion Consumer Service (ACS) URL and the Entity ID. Administrators use this information to establish a connection between their IdP server and Docker Hub.
 
 After establishing the connection between the IdP server and Docker Hub, administrators log into the organization in Docker Hub and complete the SSO enablement process. See the section **Enable SSO in Docker Hub** for detailed instructions.
 
 To enable SSO in Docker Hub, you need the following:
```
Comment (technicallyjosh) [inline single-sign-on/index.md:14]:
```suggestion
To enable SSO in Docker Hub, you need the following information from your identity provider:
```

File: single-sign-on/index.md [hunk 3]
```diff
@@ -56,12 +59,16 @@ The following video walks you through the process of configuring SSO.
 
     > **Note:**
     > the NameID is your email address and is set as the default.
-    > For example, <Subject><NameID>yourname@mycompany.com</NameID>.
+    > For example, <Subject><NameID>yourname@mycompany.com</NameID>. We also support the optional name attribute key; you must use lower-case text only.
```
Comment (technicallyjosh) [inline single-sign-on/index.md:62]:
```suggestion
    > For example, <Subject><NameID>yourname@mycompany.com</NameID>. We also support the optional `name` attribute. This attribute name must be lower-cased. _The following is an example of this attribute in Okta._
```

**Review by technicallyjosh — CHANGES_REQUESTED** [review]
Just some minor edits now.

File: single-sign-on/index.md (hunk 3 above)
Comment (jerae-duffin) [reply single-sign-on/index.md:62]: Thanks Josh!

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**